admin/config.js

```diff
@@ -57,6 +57,7 @@ module.exports = {
   SESSION_SECRET: process.env.NODE_ENV === 'production' ? process.env.SESSION_SECRET : 'anti tamper for dev',
   WEBSITE_OFFLINE: {}.hasOwnProperty.call(process.env, 'WEBSITE_OFFLINE') ? toBool(process.env.WEBSITE_OFFLINE) : false,
   WaitTimeBeforeExitInSeconds: parseInt(process.env.WAIT_TIME_BEFORE_EXIT, 10) || 30,
+  WaitTimeBeforeRedirectInSeconds: parseInt(process.env.WAIT_TIME_BEFORE_REDIRECT, 10) || 5,
```

Comment:
5 seconds seems a very long time to wait by default.

Reply:
if i recall correctly they said 2 seconds was OK?

Reply:
yes you are right, I changed it to 2